hbase-common/src/test/java/org/apache/hadoop/hbase/util/TestOrderedBytes.java

```diff
@@ -1319,4 +1320,20 @@ public void testEncodedValueCheck() {
       OrderedBytes.skip(buff);
     }
   }
+
+  @Test
+  public void testEncodeDecodeMatch() {
+    BigDecimal[] randomData = new BigDecimal[200];
+    Random rand = new Random(System.currentTimeMillis());
```

Comment:
Just use ThreadLocalRandom?

Reply:
Changed to ThreadLocalRandom and removed the redundant local variable array.